orbit/pkg/update/flag_runner_test.go

```diff
@@ -0,0 +1,59 @@
+package update
+
+import (
+	"encoding/json"
+	"reflect"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+var rawJSONFlags = json.RawMessage(`{"verbose":true, "num":5, "hello":"world"}`)
+
+func TestGetFlagsFromJson(t *testing.T) {
+	flagsJson, err := getFlagsFromJSON(rawJSONFlags)
+	require.NoError(t, err)
+
+	require.NotEmpty(t, flagsJson)
+
+	value, ok := flagsJson["--verbose"]
+	if !ok {
+		t.Errorf("key \"--verbose\" expected but not found")
```

Comment:
no need to change anything, but you can use a raw string literal (delimited by back quotes):

```suggestion
		t.Errorf(`key "--verbose" expected but not found`)
```